.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+    "singleQuote": true,
+    "tabWidth": 4,
+    "semi": false,
```

Comment:
I think we should have semicolons and we don't need to remove them.

Reply:
Okay, semicolon it is.

Reply:
I think you will need to run the prettier again so that this change is applied.

Reply:
My bad, forgot to do that